clients/remote/remote.go

```diff
@@ -0,0 +1,115 @@
+package remote
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/influxdata/influx-cli/v2/api"
+	"github.com/influxdata/influx-cli/v2/clients"
+)
+
+type Client struct {
+	clients.CLI
+	api.RemoteConnectionsApi
+	api.OrganizationsApi
+}
+
+type CreateParams struct {
+	Name             string
+	Description      string
+	OrgID            string
+	OrgName          string
+	RemoteURL        string
+	RemoteAPIToken   string
+	RemoteOrgID      string
+	AllowInsecureTLS bool
+}
+
+func (c Client) Create(ctx context.Context, params *CreateParams) error {
+
+	if params.OrgID == "" && params.OrgName == "" && c.ActiveConfig.Org == "" {
+		return clients.ErrMustSpecifyOrg
+	}
+
+	// get org id via org name
+	if params.OrgID == "" {
+		name := params.OrgName
+		if name == "" {
+			name = c.ActiveConfig.Org
+		}
+		res, err := c.GetOrgs(ctx).Org(name).Execute()
+		if err != nil {
+			return fmt.Errorf("failed to lookup ID of org %q: %w", name, err)
+		}
+		orgs := res.GetOrgs()
+		if len(orgs) == 0 {
+			return fmt.Errorf("no organization found with name %q", name)
+		}
+		params.OrgID = orgs[0].GetId()
+	}
+
+	// set up a struct with required params
+	body := api.RemoteConnectionCreationRequest{
+		Name:             params.Name,
+		OrgID:            params.OrgID,
+		RemoteURL:        params.RemoteURL,
+		RemoteAPIToken:   params.RemoteAPIToken,
+		RemoteOrgID:      params.RemoteOrgID,
+		AllowInsecureTLS: params.AllowInsecureTLS,
+	}
+
+	if params.Description != "" {
+		body.Description = &params.Description
+	}
+
+	// send post request
+	res, err := c.PostRemoteConnection(ctx).RemoteConnectionCreationRequest(body).Execute()
+	if err != nil {
+		return fmt.Errorf("failed to create remote connection %q: %w", params.Name, err)
+	}
+	// print confirmation of new connection
+	return c.printRemote(printRemoteOpts{remote: &res})
+}
+
+type printRemoteOpts struct {
+	remote  *api.RemoteConnection
+	remotes []api.RemoteConnection
+	deleted bool
+}
+
+func (c Client) printRemote(opts printRemoteOpts) error {
+	if c.PrintAsJSON {
+		var v interface{}
+		if opts.remote != nil {
+			v = opts.remote
+		} else {
+			v = opts.remotes
+		}
+		return c.PrintJSON(v)
+	}
+
+	headers := []string{"Org ID", "Remote URL", "Remote Org ID", "Allow Insecure TLS"}
```

Comment:
Oops, miscommunication: I think we want these _and_ the `ID` and `Name` headers you had here previously.